src/main/java/stsjorbsmod/memories/WrathMemory.java

```diff
@@ -94,6 +94,12 @@ private void permanentlyIncreaseCardDamage(AbstractCard card) {
             return;
         }
 
+        // ignore cards that are being removed from the deck
```

Comment:
I don't think I agree with this change; I understand it's mechanically not different, but I think showing the vfx is important so that the player understands what happened. If anything, I think it might make sense to addToTop a WaitAction to go with the wrath effect, to give the player time to process it (that might also help make the Patron+ effect chain more understandable)